packages/ui/react-ui/src/CrossMintStatusProvider.tsx

```diff
@@ -19,6 +19,11 @@ export const CrossMintStatusProvider: FC<CrossMintStatusProviderProps> = ({
     );
 
     async function fetchClientIntegration() {
+        if (!clientId || clientId === "" || clientId === "<YOUR_CLIENT_ID>") {
```

Comment:
!clientId already covers the ```clientId === ""``` use case

Reply:
Crap, I missed this comment. I'll fix on a new PR